backend/molgenis-emx2-email/src/main/java/org/molgenis/emx2/email/EmailValidator.java

```diff
@@ -0,0 +1,78 @@
+package org.molgenis.emx2.email;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import javax.mail.internet.AddressException;
+import javax.mail.internet.InternetAddress;
+import org.molgenis.emx2.MolgenisException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class EmailValidator {
+
+  private static final Logger logger = LoggerFactory.getLogger(EmailValidator.class);
+
+  public static InternetAddress toInternetAddress(String email) {
+    try {
+      return new InternetAddress(email);
+    } catch (Exception e) {
+      throw new MolgenisException("Invalid email address: " + email);
+    }
+  }
+
+  public static boolean isInValidEmail(String email) {
+    try {
+      new InternetAddress(email);
```

Comment:
we probably should merge this with existing ColumnType.EMAIL validation?

Reply:
for https://github.com/molgenis/molgenis-emx2/pull/2375 , but maybe , this way we could also send mail outside of emx2 email model